html/home.html

```diff
@@ -0,0 +1,50 @@
+<html lang="en">
+    <!--1044127873768-s5691d509qqlq1pfgjilkiq0cpkdtima.apps.googleusercontent.com-->
+    <!--a1GrL4Fp0Ka-RQ8DYIz95013-->
```

Comment:
this is dangerous, please don't check this in!
since this is already recorded in git/github, i would recommend creating a new clientID and using that instead.

Reply:
Okay. I forgot I commented that :/